core/src/main/java/knaufdan/android/core/calendar/CalendarExtensions.kt

```diff
@@ -0,0 +1,20 @@
+package knaufdan.android.core.calendar
+
+import java.util.Calendar
+
+fun Calendar.getDayOfMonth(): DayOfMonth = get(Calendar.DAY_OF_MONTH)
+
+fun Calendar.getMonth(): Month = get(Calendar.MONTH) + 1
```

Comment:
do not add `+1` here, because all other calculations might crash. Add comment instead.

Reply:
done.